setup.py

```diff
@@ -148,7 +148,12 @@
     _smoothers_lowess={'source': 'statsmodels/nonparametric/_smoothers_lowess.pyx'},  # noqa: E501
     kalman_loglike={'source': 'statsmodels/tsa/kalmanf/kalman_loglike.pyx',
                     'include_dirs': ['statsmodels/src'],
-                    'depends': ['statsmodels/src/capsule.h']})
+                    'depends': ['statsmodels/src/capsule.h']},
+    _cy_kernels={'source' : 'statsmodels/kernel_methods/_cy_kernels.pyx',
+                 'depends' : ['statsmodels/kernel_merhods/erf.h']},
```

Comment:
I think this could match the pattern in kalman_loglike, i.e. move erf.h to src, and specify include_dirs
(but I'm not an expert on building with cython)